src/profile-logic/tracks.js

```diff
@@ -261,7 +261,7 @@ export function computeLocalTracksByPid(
 
   // find markers that might have their own track.
   const markerSchemasWithGraphs = (profile.meta.markerSchema || []).filter(
-    (schema) => schema.graphs !== undefined
+    (schema) => schema.graphs !== undefined && schema.graphs.length > 0
```

Comment:
If I wanted to be super pedantic I would say `Array.isArray` might be a better check than `!== undefined` here before accessing the length. But I don't really mind.